File: bank-kyc-system/src/main/java/com/bank/project/dto/AccountInfoDTO.java
package com.bank.project.dto;

import java.math.BigDecimal;

import com.bank.project.enums.AccountStatus;
import com.bank.project.enums.AccountType;
import com.bank.project.enums.KycStatus;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class AccountInfoDTO {
    private String accountNumber;
    private AccountType accountType;
    private String branchName;
    private String ifscCode;
    private BigDecimal balance;
    private KycStatus kycStatus;   
    private AccountStatus accountStatus;
}

File: BankKycSystemdotnet/BankKycSystem/DTOs/AccountInfoDTO.cs
using BankKycSystem.Enums;

namespace BankKycSystem.DTOs
{
    public class AccountInfoDTO
    {
        public string AccountNumber { get; set; }
        public AccountType AccountType { get; set; }
        public string BranchName { get; set; }
        public string IFSCCode { get; set; }
        public decimal Balance { get; set; }
    }
}
